packages/core/__tests__/JS-node-runtime-test.ts

```diff
@@ -1,18 +1,18 @@
-/**
- * @jest-environment node
- */
-
-/** The doc block above is to change the running environment of Jest to node.
- * Since this is allowed per test file and not per test or describe, we have
- * two tests, one for node and other for browser
- */
-import * as core from '../lib';
-
-describe('JS build test', () => {
-	test('when its node ', () => {
-		expect(core.JS.browserOrNode()).toStrictEqual({
-			isBrowser: false,
-			isNode: true,
-		});
-	});
-});
+/**
+ * @jest-environment node
+ */
+
+/** The doc block above is to change the running environment of Jest to node.
+ * Since this is allowed per test file and not per test or describe, we have
+ * two tests, one for node and other for browser
+ */
+import * as core from '../lib';
+
+describe('JS build test', () => {
+	test('when its node ', () => {
+		expect(core.browserOrNode()).toStrictEqual({
```

Comment:
Nit: Could we just import what we need?
```suggestion
import { browserOrNode } from '../lib';

describe('browserOrNode build test', () => {
	test('when it\'s node ', () => {
		expect(browserOrNode()).toStrictEqual({
```

Reply:
Updated in latest diff